fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,31 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
-    g_signal_emit_by_name( getSessionCallbacksObject(), "joining", fv );
-    
+    if (cc) {
```

Comment:
I think it's meant to be called because it's signalling that we're leaving the collab session. But then again I don't know much about what the collab code is meant to do.